src/localeStorageManager.js

```diff
@@ -2,9 +2,22 @@ import isEqual from "lodash.isequal";
 
 const STORAGE_KEY = "hunger-game-settings";
 
+export const localSettingsKeys = {
+  language: 'lang',
+  isDevMode: 'devMode',
+  hideImages: 'questions_hideImages',
+}
+
+
 export const localSettings = {
   fetch: function () {
-    return JSON.parse(localStorage.getItem(STORAGE_KEY) || "{}");
+    let storedValue = {};
+    
+    try{
```

Comment:
In which case could it faill?

Reply:
the issue is that as you can  mess with the local storage it can have something that raises an error because is invalid, like the string "{a:[22"